ema/nextjs/src/components/dotcms-page.js

```diff
@@ -1,9 +1,21 @@
 'use client';
 
-import { useContext } from 'react';
+import { useContext, useEffect } from 'react';
 import { GlobalContext } from '../providers/global';
 
+function reloadWindow(event) {
+    if (event.data !== 'reload') return;
+
+    window.location.reload();
+}
+
 function WebPageContent({ title, body }) {
+    useEffect(() => {
+        window.removeEventListener('message', reloadWindow);
+
+        window.addEventListener('message', reloadWindow);
+    }, []);
```

Comment:
Check if you can use `_app.js` in nextjs.

Reply:
Did a different approach